packages/api/lib/ems.js

```diff
@@ -101,6 +103,22 @@ async function getExpiredS3Objects(bucket, prefix, retentionInDays) {
     .map((s3Object) => ({ Bucket: bucket, Key: s3Object.Key }));
 }
 
+async function retrievePrivateKey() {
+  const privateKeyFile = process.env.ems_privateKey || 'ems-private.pem';
+
+  const keyExists = await s3ObjectExists(
```

Comment:
Rather than have an explicit check to see if the key exists, can you just call `getS3Object`, catch an exception if it's thrown, and throw an "object does not exist" error? That would save an API call that isn't really needed.

Reply:
Ok changed it.